ballerina/tests/tests.bal

```diff
@@ -0,0 +1,85 @@
9,135,109,195,135,113,100,215,75,237,24,131,147,238,229,62,18,194,208,41,49,61,191,151,113,62,148,123,24,206,169,91,20,134,55,236,192,23,31,58,137,47,239,225,98,140,4,207,195,52,110,99,249,246,74,124,67,7,193,91,195,31,195,4,109,35,62,185,235,26,78,80,119,92,146,164,19,95,120,240,228,205,155,119,254,12,235,206,187,0,218,91,99,100,195,78,60,68,198,177,249,190,79,123,4,118,19,171,24,82,222,56,253,17,30,251,90,6,95,125,106,115,34,121,248,225,252,9,97,187,68,57,141,225,229,233,11,33,12,150,101,89,207,42,239,225,97,60,177,42,80,128,239,20,215,31,229,94,223,126,115,9,55,174,221,197,197,215,45,28,249,22,193,198,225,85,165,254,224,202,199,120,112,239,80,96,191,248,140,151,2,169,148,47,139,238,121,203,248,5,105,255,185,117,186,170,1,73,207,47,120,232,175,222,0,5,176,124,107,5,243,191,92,245,100,87,46,45,227,167,207,127,24,34,67,244,248,81,219,168,239,132,172,78,234,93,162,120,236,31,229,18,25,254,112,97,9,215,63,188,235,182,185,248,186,229,218,146,11,103,111,226,250,181,187,120,251,205,165,160,201,59,204,120,87,137,25,128,134,240,229,112,109,241,8,20,130,243,175,222,112,251,1,171,90,114,229,210,50,254,242,206,178,167,141,79,134,174,142,2,232,96,52,196,63,97,60,49,34,126,127,97,9,31,93,187,235,235,115,241,117,11,111,188,124,221,157,227,70,136,150,120,2,47,159,60,225,196,244,95,67,34,109,70,180,125,225,193,108,135,56,198,191,150,86,208,124,247,223,220,152,193,90,194,223,15,210,22,94,62,254,122,95,189,140,71,16,103,134,84,42,192,205,58,87,82,206,21,209,237,222,184,118,23,143,77,120,115,162,48,79,196,174,223,184,118,23,91,182,175,119,198,23,231,115,234,140,252,136,51,216,190,19,226,115,115,116,213,245,198,17,115,249,183,183,113,240,75,195,216,178,125,61,212,132,49,135,136,43,151,150,61,178,120,231,115,234,1,196,112,113,74,76,84,212,33,158,206,93,113,7,140,250,28,193,31,54,167,112,255,72,26,79,190,240,0,54,110,74,119,156,250,3,109,163,251,218,139,255,192,178,147,254,119,147,241,246,157,144,111,231,174,208,88,65,98,136,73,222,54,126,252,168,126,190,122,106,64,91,70,156,200,167,186,93,216,151,224,182,94,4,141,31,56,63,188,164,184,117,58,32,163,26,42,168,88,95,3,98,66,101,113,254,15,196,237,186,101,225,199,95,143,235,179,90,239,61,21,72,210,135,97,48,129,89,72,224,21,37,72,92,20,27,68,76,120,219,238,137,25,108,232,30,32,64,220,19,138,139,98,59,73,5,162,136,9,235,115,111,66,247,30,137,233,100,27,1,65,36,250,175,135,145,57,208,192,44,137,17,133,216,38,65,159,65,26,222,129,121,153,64,183,198,9,33,186,68,134,181,38,102,128,231,33,221,217,138,48,117,231,251,133,207,209,187,71,26,76,182,27,48,113,104,125,192,134,151,181,143,147,103,176,94,134,222,123,98,186,49,188,226,216,192,0,189,12,43,196,46,74,172,175,161,71,26,136,134,0,104,121,4,115,10,73,136,225,145,52,226,237,7,49,92,219,165,65,108,153,239,6,10,205,17,35,180,247,183,101,245,16,53,79,66,76,232,189,8,98,62,161,244,251,125,39,228,181,223,237,253,25,128,167,41,208,10,35,166,23,251,2,238,94,36,185,177,54,195,67,204,210,10,165,197,159,24,123,181,255,2,134,1,253,246,81,242,86,147,0,0,0,0,73,69,78,68,174,66,96,130];   
```

Comment:
Instead of repeating the byte array we can assign it to a module-level variable and re-use it

Reply:
tried to define the `sampleImageContent` with `final`. but it did not allow to use inside the isolated function. 
came up with the error -> invalid access of mutable storage in an 'isolated' function(BCE3943)

Reply:
https://stackoverflow.com/questions/78852035/invalid-access-of-mutable-storage-in-an-isolated-ballerina-function-despite-usin

Reply:
Replied to that. You can simply use the `readonly` type to overcome this error